app/routers/entitlements.py

```diff
@@ -27,6 +29,6 @@ async def create_entitlement(data: EntitlementCreate, session: DBSession):
 
 
 @router.patch("/{id}", response_model=EntitlementRead)
```

Comment:
Apologize I miss it in the previous PR, there is no patch for entitlements, please check the TDR article. Thank you!

Reply:
Removed the endpoint and its tests :)